dvc/dvcfile.py

```diff
@@ -194,6 +194,9 @@ def _dump_pipeline_file(self, stage):
             with open(self.path) as fd:
                 data = parse_stage_for_update(fd.read(), self.path)
         else:
+            logger.info(
+                "%s does not exist, creating…", styled(self.relpath, "bold")
```

Comment:
Minor: Do we use `…` (ellipsis character) a lot? I'm guessing it's mostly `...` (3 period chars) in other places, if any.

Reply:
Also
```suggestion
                "'%s' does not exist, creating…", styled(self.relpath, "bold")
```